openpgp/read_write_test_data.go

```diff
@@ -455,3 +455,697 @@ byVJHvLO/XErtC+GNIJeMg==
 =liRq
 -----END PGP MESSAGE-----
 `
+
+// Generated with the above private key
+const pgpPhotoPublicOpenPGP2_2_34 = `-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
```suggestion
const pgpPhotoPublicGPG2_2_34 = `-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Also, you haven't used this one AFAICT - is it an oversight?